lib/src/result.dart

```diff
@@ -37,7 +37,7 @@ abstract class Result<T> {
   /// The result of the transformation is a sink that only forwards [Result]
   /// values and no error events.
   static const StreamSinkTransformer<Object, Result> captureSinkTransformer =
-      const StreamSinkTransformer.fromStreamTransformer(
```

Comment:
(the fact that the type annotation is required looks like some sort of bug to me, but of what sort or whether we have it filed already, I'm not certain)